src/components/tx-index/IndexMsgIcon.re

```diff
@@ -28,7 +28,7 @@ let make = (~category: MsgDecoder.msg_cat_t) => {
      | ValidatorMsg => <Icon name="fas fa-user" color=Theme.white size=14 />
      | ProposalMsg => <Icon name="fal fa-file" color=Theme.white size=14 />
      | DataMsg => <Icon name="fal fa-globe" color=Theme.white size=14 />
-     | _ => <Icon name="far fa-wallet" color=Theme.white size=14 />
+     | _ => <img src=Images.ibcIcon />
```

Comment:
IMO, we should handle IBC messages by using `ibcIcon` and handle others by using `unknown` or some default icon.